crates/turborepo-lib/src/turbo_json/mod.rs

```diff
@@ -100,27 +157,27 @@ impl DerefMut for Pipeline {
     }
 }
 
-#[derive(Serialize, Deserialize, Default, Debug, PartialEq, Clone)]
+#[derive(Serialize, Default, Debug, PartialEq, Clone)]
 #[serde(rename_all = "camelCase")]
 pub struct RawTaskDefinition {
     #[serde(skip_serializing_if = "Option::is_none")]
-    cache: Option<bool>,
+    cache: Option<Spanned<bool>>,
```

Comment:
For these fields does it make sense to flip the types? e.g. `Spanned<Option<bool>>`
If a field isn't declared we still might want to to reference something like "no inputs declared in path/to/turbo.json"

Reply:
What would the span be in the case where we don't have a field?